packages/manager/src/features/VPCs/VPCCreate/SubnetNode.tsx

```diff
@@ -52,49 +61,55 @@ export const SubnetNode = (props: Props) => {
     onChange(subnet, idx, isRemovable);
   };
 
+  const showRemoveButton = isCreateVPCDrawer
+    ? idx !== 0 && isRemovable
+    : isRemovable;
+
   return (
     <Grid key={idx} sx={{ maxWidth: 460 }}>
       <Grid container direction="row" spacing={2}>
         <Grid
-          sx={{ ...(!isRemovable && { width: '100%' }) }}
-          xs={isRemovable ? 11 : 12}
+          sx={{ ...(!showRemoveButton && { width: '100%' }), flexGrow: 1 }}
+          xs={showRemoveButton ? 11 : 12}
         >
-          <TextField
-            disabled={disabled}
-            errorText={subnet.labelError}
-            inputId={`subnet-label-${idx}`}
-            label="Subnet Label"
-            onChange={onLabelChange}
-            placeholder="Enter a subnet label"
-            value={subnet.label}
-          />
+          <Stack>
+            <TextField
+              aria-label="Enter a subnet label"
+              disabled={disabled}
+              errorText={subnet.labelError}
+              inputId={`subnet-label-${idx}`}
+              label="Subnet Label"
+              onChange={onLabelChange}
+              placeholder="Enter a subnet label"
+              value={subnet.label}
+            />
+            <TextField
+              aria-label="Enter an IPv4"
+              disabled={disabled}
+              errorText={subnet.ip.ipv4Error}
+              inputId={`subnet-ipv4-${idx}`}
+              label="Subnet IP Address Range"
+              onChange={onIpv4Change}
+              value={subnet.ip.ipv4}
+            />
+            {subnet.ip.availIPv4s && (
+              <FormHelperText>
+                Number of Available IP Addresses:{' '}
+                {subnet.ip.availIPv4s > 4
+                  ? (subnet.ip.availIPv4s - RESERVED_IP_NUMBER).toLocaleString()
+                  : 0}
+              </FormHelperText>
+            )}
+          </Stack>
         </Grid>
-        {isRemovable && (
+        {showRemoveButton && (
           <Grid xs={1}>
-            <StyledButton onClick={removeSubnet} aria-label="Remove Subnet">
+            <StyledButton aria-label={`Remove Subnet`} onClick={removeSubnet}>
```

Comment:
Did the linter change this line?

```suggestion
            <StyledButton aria-label="Remove Subnet" onClick={removeSubnet}>
```

Reply:
Think so!